### ISSUE TITLE
MPU6050_DMP6 - would it run on Intel Galileo v1 with Windows 10 on it?

### ISSUE THREAD

**Issue by marwol18** [jrowberg/i2cdevlib#163]

**Comment by marwol18** [comment]
Hi guys,
It's me again. It turns out, this library and the sample DOES WORK on the Intel Galileo running Windows for IOT.

The code has to be modified, though. Interrupts don't work. The attachInterrupyt) methods can be dropped altogether. What works is a constant checking of the InerruptStatus Register on the MPU6050 in the loop(). That's all. One thing to note is that the FIFO fills up really fast. If your code is not neat, you'll be getting frequent FIFO Overflows.

**Comment by marwol18** [comment]
I forgot to mention some compiler issues. The compiler complaned about some stuff. A couple of manual corrections in the code did the job.

**Comment by marwol18** [comment]
OK, a new problem. The Sample MPU6050_DMP6 does work after some minor modifications. A new problem arises. This one is harder to figure out. The porgram runs just fine and then unexpectedly exits with the following message:

Sketch Aborted! A fatal error has occurred:
Error encountered performing queued I2C transfers to address: 0x68, Error: 0x000002C3
The program '[1400] MPU6050_DMP.exe' has exited with code 1 (0x1).

I experimented with a couple of things to try to single out the fault. 
1. I added a delay() to the loop(). In general, the longer the delay period, the longer it runs before crashing (but I need a sample rate of at least 30HZ). As an example, a delay(30) lets it run for about a minute on average before crashing.
2. The more comms are used the sooner the program crashes. For exmaple, the Serial causes trouble. Even standard c++ std:: cout to console over telnet causes trouble.

I wonder, if it's got something to do with the library or the platform itself? I heard that the performance of this Galileo board is not the best. Would it be it? And of course there is the Windows OS.
